git_machete/docs.py

```diff
@@ -662,6 +662,24 @@
 
           <b>--no-detect-squash-merges</b>         Only consider "strict" (fast-forward or 2-parent) merges, rather than rebase/squash merges,
                                             when detecting if a branch is merged into its upstream (parent).
+
+        To make it easier to select branch name from the `status's` output on certain terminals (E.g. Alacritty), you can add additional
+        space between └─ and branch name by setting git config key `machete.status.extraSpaceBeforeBranchName` = `true`.
+
+        E.g.
+            default:
+              develop
+              │
+              ├─feature_branch1
+              │
+              └─feature_branch2
+
+            with extra space:
+               develop
+               │
+               ├─ feature_branch1
+               │
+               └─ feature_branch2
```

Comment:
Formatting suggestion.
Also a small observation from my experience: I've been asked what `e.g.` means in the past, so nowadays I assume it's not widely used and I try to avoid it in documentation.

```suggestion
        space between └─ and branch name by setting `git config machete.status.extraSpaceBeforeBranchName true`.
        For example, by default it's:

              develop
              │
              ├─feature_branch1
              │
              └─feature_branch2

        With the `extraSpaceBeforeBranchName` option set:
            
               develop
               │
               ├─ feature_branch1
               │
               └─ feature_branch2
```

Reply:
Makes sense... lemme actually open an issue to ban all `e.g.`s and `i.e.`s from the docs as they're notoriously confused with each other